certified-connectors/SoloSign/Readme.md

```diff
@@ -0,0 +1,75 @@
+# SoloSign API Documentation
+
```

Comment:
Hello @cueup, 
please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.

Reply:
I've updated the readme file and pushed the update